hudi-hadoop-common/src/test/java/org/apache/hudi/common/testutils/FileSystemTestUtils.java

```diff
@@ -79,34 +68,4 @@ public static void deleteFile(File fileToDelete) throws IOException {
       throw new IOException(message);
     }
   }
-
-  public static List<FileStatus> listRecursive(FileSystem fs, Path path) throws IOException {
-    return listFiles(fs, path, true);
-  }
-
-  public static List<FileStatus> listFiles(FileSystem fs, Path path, boolean recursive) throws IOException {
-    RemoteIterator<LocatedFileStatus> itr = fs.listFiles(path, recursive);
-    List<FileStatus> statuses = new ArrayList<>();
-    while (itr.hasNext()) {
-      statuses.add(itr.next());
-    }
-    return statuses;
-  }
-
-  public static List<StoragePathInfo> listRecursive(HoodieStorage storage, StoragePath path)
-      throws IOException {
-    return listFiles(storage, path);
-  }
-
-  public static List<StoragePathInfo> listFiles(HoodieStorage storage, StoragePath path)
-      throws IOException {
-    return storage.listFiles(path);
-  }
-
-  public static String readLastLineFromResourceFile(String resourceName) throws IOException {
-    try (InputStream inputStream = TestLogReaderUtils.class.getResourceAsStream(resourceName)) {
-      List<String> lines = FileIOUtils.readAsUTFStringLines(inputStream);
-      return lines.get(lines.size() - 1);
-    }
-  }
```

Comment:
Are these only used by tests?  Should the methods stay here if there is no compelling reason to move them?

Reply:
```
  public static String readLastLineFromResourceFile(String resourceName) throws IOException {
    try (InputStream inputStream = TestLogReaderUtils.class.getResourceAsStream(resourceName)) {
      List<String> lines = FileIOUtils.readAsUTFStringLines(inputStream);
      return lines.get(lines.size() - 1);
    }
  }
```
needed to be moved since it uses a class that's now in hudi-hadoop-common. Since I was moving that one, I just moved all the rest of the hadoop ones as well

Reply:
add info to pr description